packages/xo-web/src/xo-app/home/host-item.js

```diff
@@ -41,6 +41,11 @@ import styles from './index.css'
   ),
 }))
 export default class HostItem extends Component {
+  componentWillMount () {
+    subscribeHostMissingPatches(this.props.item, missingPatches =>
+      this.setState({ missingPatches })
+    )
+  }
```

Comment:
Use `addSubscriptions` instead. You don't need to put `missingPatches` in the `state`.